apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableStatsCollectorUtil.java

```diff
@@ -236,4 +244,58 @@ private static long getSumOfFileSizeBytes(Dataset<Row> allDataFiles) {
         .first()
         .getLong(0);
   }
+
+  /**
+   * Get sharingEnabled from table policies.
+   *
+   * @param table
+   * @return
+   */
+  private static Boolean getTablePoliciesSharingEnabled(Table table) {
+    String policies = table.properties().get("policies");
+    if (policies.isEmpty()) {
+      return false;
+    }
+
+    Boolean sharingEnabled =
+        Boolean.valueOf(
+            new Gson().fromJson(policies, JsonObject.class).get("sharingEnabled").getAsString());
+
+    return sharingEnabled;
+  }
+
+  /**
+   * Get retention policy from table policies.
+   *
+   * @param table
+   * @return
+   */
+  private static RetentionStatsSchema getTablePolicyRetention(Table table) {
+    String policies = table.properties().get("policies");
```

Comment:
Can extracting the policies from both these methods be de-duped?

Reply:
changed function to put all policies in its own map